_docs/other/developer-guide.md

```diff
@@ -3,14 +3,18 @@ title: Developer Guide
 ---
 
 ## Compiling
-1. Clone the repo to your computer (EG: `git clone https://github.com/GeyserMC/Geyser.git`)
-2. [Install Maven](https://maven.apache.org/install.html)
-3. Navigate to the Geyser root directory and run `git submodule update --init --recursive`. This downloads all the needed submodules for Geyser and is a crucial step in this process.
-4. Run `mvn clean install` and locate to the `bootstrap`, then your desired Geyser version, then `target` folder.
+1. Clone the repo to your computer.
+2. Navigate to the Geyser root directory and run `git submodule update --init --recursive`. This command downloads all the needed submodules for Geyser and is a crucial step in this process.
+3. Run `gradlew build` and locate to `bootstrap/<platform>/build/` folder.
 
 ## Project layout
+
+Geyser's code is divided into different modules. For example:
+
 * `bootstrap` is where we hold the specific platform code. So if you're porting Geyser to a new platform, or working with platform-specific code, you likely want to be in here.
-* `core` is where connections are handled and the data conversion is done. The majority of Geyser work is held here.
+* `core` is where connections are handled and the data/packet conversion is done. The majority of Geyser work is held here.
+* `api` is where the Geyser API is found.
+* `build-logic` contains the kotlin gradle plugins that are used to build Geyser.
```

Comment:
The wording is inconsistent here, "is where" to "contains"

Reply:
Thought I'd mix it up to make it less repetitive, but sure, I'll revert

Reply:
Either works, this isn't really an academic essay that would require breaking up the flow tho 😄